examples/where_is_my_output/README.md

```diff
@@ -0,0 +1,66 @@
+# Where are my files?
+
+## Finding the exact path to the files created by a target.
+
+Most of the time, Bazel users do not need to know the path to the artifacts
+created for any given target. A notable exception is for users of packaging
+rules. You typically create an RPM or Debian packaged file for the explicit
+purpose of taking it from your machine and giving it to someon else.
+
+Users often create scripts to push `bazel build` outputs to other places and
+need to know the path to those outputs. This can be a challenge for rules which
+may create the file name by combining a base part with a version number,
+and maybe a CPU architecture. We don't do find them with shell wildcards
+like `bazel-bin/my-pkg/pkg-*.deb`. That is brittle. Fortunately, Bazel
+provide all the tools we need to get the pricise path to an output.
+
+## Using cquery to find the exact path to the outputs created for a target.
+
+We can use Bazel's cquery command to find information about a target.
+Specifically we use
+[cquery's Starlark output](https://docs.bazel.build/versions/main/cquery.html#cquery-starlark-dialect)
+to inspect a target and print exactly what we need. Let's try it:
```

Comment:
The document might flow better if you separate the URL from the link specification.  So, something like:

```markdown
Specifically we use [cquery's Starlark output] ...

...later...
[cquery's Starlark output]: https://docs.bazel.build/versions/main/cquery.html#cquery-starlark-dialect
```

I don't know if github-flavored markdown supports this, though.

Reply:
I'm going to hold off on new flow ideas for now. There is a good chance we will get some professional tech writing resources to create a holistic style guide for all of Bazel's docs, tutorials and examples. Once that is set, we can do a large fixit pass